### ISSUE TITLE
Script variant and kitty

### ISSUE THREAD

**Issue by chmanie** [MonoLisaFont/feedback#160]
## Background information

To help debugging, fill the following information before describing the issue:

* Font edition (i.e. trial, personal, plus, pro): Plus
* Font version (i.e. 1.805): 1.808
* The type of the font file you are using (i.e. otf, ttf): otf
* Operating system (i.e. MacOS, Windows): Linux
* Display resolution (i.e. 3072x1920): 1920x1080
* Any other information that might be helpful: kitty 0.25.2

## Issue

I am trying the newest version of the Plus font (1.808) and I can't get it to work with kitty.

This is my current config:
```conf
font_family      MonoLisa
bold_font        MonoLisa Medium
italic_font      MonoLisa
bold_italic_font MonoLisa Medium
```
```
| kitty +list-fonts --psnames | rg MonoLisa
MonoLisa
    MonoLisa Bold (MonoLisa-Bold)
    MonoLisa Light (MonoLisa-Light)
    MonoLisa Medium (MonoLisa-Medium)
    MonoLisa Regular (MonoLisa-Regular)
```

 I have all sorts of problems:
- The italic font is not italic anymore (that is after an upgrade to 1.808)
- I tried to use kitty's `font_features`, and sometimes they worked and then (I don't really think there should be a difference in the settings) they didn't work anymore (e.g. `+zero` worked for me briefly, but when I add it now, it does not)
- The `+script` feature never worked for me, not even when downloading the customized version of the font.

Long story short, does anyone have a working kitty config, so that I can maybe reproduce it somehow and see what the issue might be here?

Thank you so much.

**Comment by chmanie** [comment]
Hm. I wonder why the `Italic` styles don't show up for me. They're in the `fonts` folder:

```
.fonts/MonoLisa
| ls
MonoLisa-Bold.otf        MonoLisa-LightItalic.otf   MonoLisa-Regular.otf
MonoLisa-BoldItalic.otf  MonoLisa-Medium.otf        MonoLisa-RegularItalic.otf
MonoLisa-Light.otf       MonoLisa-MediumItalic.otf
```
